packages/appium/docs/en/guides/migrating-1-to-2.md

```diff
@@ -194,6 +194,16 @@ In summary, if you are using a JSON Appium config file, you can simply cut-and-p
 
 The old iOS and Android (UiAutomator 1) drivers and related tools (e.g., `authorize-ios`) have been removed. They haven't been relevant for many years anyway.
 
+### :bangbang: Server can no longer be started with `--port 0`
+
+In Appium 1.x, it was possible to specify `--port 0` during server startup. This had the effect of
+starting Appium on a random free port. In Appium 2, port values must be `1` or higher. The random
```

Comment:
```suggestion
starting Appium on a random free port. In Appium 2.0, port values must be `1` or higher. The random
```

[nit] Current this documentation addresses appium 2 as `2.0`, so 2.0 would be more consistent